brainstormforce/astra
# feat: Toggle on Desktop. ( Off-Canvas & Dropdown Support )

### Description
 - Trello task - https://trello.com/c/kcWKZN7H/871-toggle-for-desktop

### Types of changes
New feature (non-breaking change which adds functionality)

### How has this been tested?
 - Check Backward compatibility.
 - Check by adding/removing Toggle the Button element in Desktop Header.

### Checklist:
